modules/developer-cli-odo-basic-odo-cli-commands.adoc

```diff
@@ -360,6 +360,9 @@ Modify `odo` specific configuration settings within the global preference file.
 
   # Disable experimental mode
   odo preference set experimental false
+  
+  # Use ephemeral volumes in the cluster
+  odo preference set ephemeral true
```

Comment:
Shouldn't this example about setting the value to `false` since the default is `true`? Or am I looking it the wrong way? :thinking: